checkers/postgresql.go

```diff
@@ -19,9 +19,15 @@ package checkers
 import (
 	"context"
 	"database/sql"
+	"time"
 )
 
 // PostgreSQL checks whether PostgreSQL server is primary or not.
 func PostgreSQL(ctx context.Context, db *sql.DB) (bool, error) {
 	return Check(ctx, db, "SELECT NOT pg_is_in_recovery()")
 }
+
+// PostgreSQLReplicationLag returns replication lag value for PostgreSQL replica node.
+func PostgreSQLReplicationLag(ctx context.Context, db *sql.DB) (time.Duration, error) {
+	return ReplicationLag(ctx, db, "SELECT NOW() - pg_last_xact_replay_timestamp()")
```

Comment:
Unfortunately on low-activity cluster this would lead to marking all replicas as lagging.
We should probably consider using [pg_stat_replication](https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-REPLICATION-VIEW) on primary with filter on `reply_time`.